### ISSUE TITLE
map.scrollWheelZoom.disable() broken in 0.10

### ISSUE THREAD

**Issue by meetar** [tangrams/tangram#427]
In https://github.com/tangrams/tangram-frame/blob/gh-pages/main.js#L238-L240 and https://github.com/tangrams/terrain-demos/blob/gh-pages/main.js#L66-L68, the `noscroll` parameter triggers `map.scrollWheelZoom.disable()` to prevent scrollWheel events.

This function no longer works with Tangram 0.10 (though it is still supported in Leaflet 1.0.1 – you can test with the map on http://leafletjs.com/ ) which makes reading posts with many embedded maps a bit tricky: https://mapzen.com/blog/mapping-mountains/

Compare:

https://tangrams.github.io/tangram-frame/?noscroll&lib=0.9&quiet&url=https://raw.githubusercontent.com/tangrams/terrain-demos/gh-pages/styles/imhof.yaml

and:

https://tangrams.github.io/tangram-frame/?noscroll&lib=0.10&quiet&url=https://raw.githubusercontent.com/tangrams/terrain-demos/gh-pages/styles/imhof.yaml

**Comment by dmvaldman** [comment]
By "this function no longer works..." do you mean "passing the option no longer suppresses scroll events"? If that's the case, that makes sense as we call `map.scrollWheelZoom.enable()` when we monkeypatch Leaflet's scroll wheel handlers, regardless of a `noscroll` option. This would re-enable Leaflet's scroll events if they internally disabled them.

**Comment by meetar** [comment]
Yes, calling `map.scrollWheelZoom.disable()` with a Tangram 0.10 map no longer disables the scrollWheelZoom, no matter when it is called.

**Comment by meetar** [comment]
Looks like this is broken only in v10.0.5, it works in master as of https://github.com/tangrams/tangram/commit/63bd1b75cd6c866ce547e564db2078c1511a5fc3. Will close this after verifying on next release.

**Comment by dmvaldman** [comment]
Verified that there is a related bug if (rare case) you initialize the Leaftlet layer with the options `scrollWheelZoom : false` as in

```
Tangram.leafletLayer({
    scrollWheelZoom: false
    ...
});
```

**Comment by bcamper** [comment]
Oh sorry I probably broke this, I'll take a careful look to make sure the user's existing enable/disable state is respected, and that it can also be changed.

**Comment by bcamper** [comment]
Resolved in https://github.com/tangrams/tangram/commit/82e4e76832a6bdac2d6f41d846c3367ec6dd171c

Close issue on release & verification.

**Comment by bcamper** [comment]
Also adding an intermediary fix for v0.10.6 in https://github.com/tangrams/tangram/commit/a07c11e148577f3cc04188d1335c215cc8ef3327

**Comment by bcamper** [comment]
Fix released in https://github.com/tangrams/tangram/releases/tag/v0.10.6
